pkg/prom/exporter_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prom
+
+import (
+	"fmt"
+
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"gotest.tools/assert"
+)
+
+var (
+	staticLabels = map[string]string{"builder": "test"}
+	builder      = NewBuilder("onos", "exporter_test", staticLabels)
+)
+
+const (
+	address = "localhost:9999"
```

Comment:
what about this address? Is there any chance this address can be something other than localhost?

Reply:
This is just for testing, it creates a prometheus exporter (http server) serving on this address:port by default.
When creating a NewExporter this value must be provided.